src/node.cc

```diff
@@ -4202,17 +4201,19 @@ inline void PlatformInit() {
 
     s.flags = GetFileDescriptorFlags(fd);
     CHECK_NE(s.flags, -1);
+    
+    if (isatty(fd)) {
```

Comment:
You could just `if (isatty(fd)) continue;` - saves a level of indent and makes the diff less noisy.

Reply:
I think, it should be `if ( ! isatty(fd)) continue;`

Reply:
That's what I mean, of course. :-)